### PR TITLE
feat: up orbitdb v0.22

### PR DESCRIPTION
**Pull request by zachferland**

### CODE REVIEW

**Comment by zachferland** [comment]
More testing necessary, and want to get version up in pinning staging sooner than later

**Comment by zachferland** [comment]
This will perform an indexeddb migration, for orbit cache

File: src/replicator.js [hunk 1]
```diff
@@ -84,7 +87,9 @@ class Replicator {
 
   async _init (opts) {
     this._pubsub = new Pubsub(this.ipfs, (await this.ipfs.id()).id)
-    this._orbitdb = await OrbitDB.createInstance(this.ipfs, { directory: opts.orbitPath })
+    // Identity not used, passes ref to 3ID orbit identity provider
+    const identity = await Identities.createIdentity({ id: 'nullid' })
+    this._orbitdb = await OrbitDB.createInstance(this.ipfs, { directory: opts.orbitPath, identity })
```
Comment (oed) [inline src/replicator.js]: Hm, shouldn't `createInstance` create an identity automatically? I don't get why this is needed.

**Comment by zachferland** [comment]
All set , and did a bunch of additional testing

**Pull request merged**